sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -373,32 +381,45 @@ export class StudentController extends BaseController {
   }
 
   /**
-   * Gets a student file validating if the user has the access to it.
+   * Gets a student file validating if the student has access to it or if the user is Aest.
    * @param userToken authentication token.
    * @param uniqueFileName unique file name (name+guid).
    * @param response file content.
    */
+  @AllowAuthorizedParty(AuthorizedParties.student, AuthorizedParties.aest)
   @Get("files/:uniqueFileName")
   async getUploadedFile(
     @UserToken() userToken: IUserToken,
     @Param("uniqueFileName") uniqueFileName: string,
     @Res() response: Response,
   ) {
-    const student = await this.studentService.getStudentByUserId(
-      userToken.userId,
-    );
+    let studentFile = undefined;
+    if (
+      AuthorizedParties.aest === userToken.authorizedParty &&
+      userToken.groups?.some(
+        (group) => UserGroups.AESTUser.toString() === group,
```

Comment:
I believe that we can omit the "toString" in this case, since the enum is a string enum.

Reply:
Feedback comment addressed.